### ISSUE TITLE
Make newer versions of rbvmomi usable

### ISSUE THREAD

**Issue by mkesper** [fog/fog-vsphere#185]
rbvmomi is released in version 2.0, changing from deprecated trollop to optimist package.

**Comment by chris1984** [comment]
@mkesper I only see v.1.12.0 out

https://github.com/vmware/rbvmomi/releases

We are using 1.9
https://github.com/fog/fog-vsphere/blob/master/fog-vsphere.gemspec#L25

Where are you seeing 2.0?

**Comment by mkesper** [comment]
Let me rephrase it: Please support rbvmomi >= 2.0
It's available as gem https://rubygems.org/gems/rbvmomi

**Comment by chris1984** [comment]
Thanks, I will move this to the planning board.

**Comment by timogoebel** [comment]
Fixed in GH-187.
